src/AnnotationDialog.js

```diff
@@ -437,17 +427,15 @@ class AnnotationDialog extends EventEmitter {
      * @private
      * @return {void}
      */
-    cancelAnnotation() {
-        this.emit('annotationcancel');
-    }
+    cancelAnnotation = () => this.emit('annotationcancel');
```

Comment:
What are the benefits of converting these functions from function declaration to fat arrow functions? Is this something we would want to do for other functions as well?

Reply:
I needed to bind these methods. I'm getting rid of a lot of these in the next PR https://github.com/box/box-annotations/pull/235